### PR TITLE
Update investigate-files.md

### PR DESCRIPTION
**Pull request by MrOS19**
Updated the changes in file page which are now GA (Oren Saban - Microsoft)

### CODE REVIEW

File: microsoft-365/security/defender-endpoint/investigate-files.md [hunk 1]
```diff
@@ -73,13 +73,16 @@ The file prevalence card shows where the file was seen in devices in the organiz
...
-:::image type="content" source="images/atp-file-information.png" alt-text="The file information" lightbox="images/atp-file-information.png":::
 
-## Alerts
+![image](https://user-images.githubusercontent.com/96785904/200525998-e11576f7-e495-4d16-98fb-940d8bd9a0d6.png)
+
 
-The **Alerts** tab provides a list of alerts that are associated with the file, as well as the incident the alert is linked to. This list covers much of the same information as the Alerts queue, except for the device group, if any, the affected device belongs to. You can choose what kind of information is shown by selecting **Customize columns** from the toolbar above the column headers.
+## Incidents and alerts
+
+The **Incidents and alerts** tab provides a list of incidents that are associated with the file, as well as the alerts  the file is linked to. This list covers much of the same information as the incidents queue. You can choose what kind of information is shown by selecting **Customize columns** from the toolbar above the column headers.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/investigate-files.md]:
```suggestion
The **Incidents and alerts** tab provides a list of incidents that are associated with the file, as well as the alerts the file is linked to. This list covers much of the same information as the incidents queue. You can choose what kind of information is shown by selecting **Customize columns** from the toolbar above the column headers.
```

File: microsoft-365/security/defender-endpoint/investigate-files.md [hunk 2]
```diff
@@ -108,6 +111,9 @@ The **File names** tab lists all names the file has been observed to use, within
 
 The **Action center** displays the action center filtered on a specific file, so you can see pending actions and the history of actions taken on the file.
 
+![image](https://user-images.githubusercontent.com/96785904/200527287-0c09dd24-6192-4a7d-990a-824d35b97460.png)
+
+
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/investigate-files.md:115]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @MrOS19, kindly commit the suggested changes. Also, note that we can't link to images outside the repo.

**Review by denisebmsft — APPROVED** [review]
Thank you @MrOS19 @JohanFreelancer9

**Pull request merged**